src/ResourceManager/PolicyInsights/Commands.PolicyInsights.Test/packages.config

```diff
@@ -13,8 +13,8 @@
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.10" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
Microsoft.Rest.ClientRuntime, Microsoft.Rest.ClientRuntime.Azure and Newtonsoft.Json shouldn't be in the .csproj or packages.config files since they are referenced elsewhere.

For example, every .csproj file should have a reference to the following file at the bottom of [their file](https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets)

Here's an example of a .csproj file referencing [this file]( https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Cdn/Commands.Cdn/Commands.Cdn.csproj#L165)